packages/vx-grid/src/types.ts

```diff
@@ -1,13 +1,21 @@
-import { D3Scale } from '@vx/scale';
+import { D3Scale, NumberLike } from '@vx/scale';
+
+// In order to plot values on an axis, output of the scale must be number.
+// Some scales return undefined.
+export type GridScaleOutput = number | NumberLike | undefined;
```

Comment:
updated this to match `AxisScaleOutput` because it simplifies the types in this PR (and logically they should be the same)